isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

```diff
@@ -14,59 +14,62 @@ Diese technische Komponente besteht aus zwei Teilen: dem Service-Framework und d
 
 Das *Service-Framework* bezeichnet die konkrete Technologie, mit der die Services des Anwendungskerns zur Verfügung gestellt werden.
 
-Die *Servicelogik* enthält Daten und Funktionalität, die für die Bereitstellung des Services relevant sind.
-
+Die *Servicelogik* gliedert sich, wie der Anwendungskern, in Fachkomponenten.
```

Comment:
Zeile 17 (alt) würde ich so belassen und deine Ergänzungen (Zeile 17 (neu) - 20) in das Kapitel Servicelogik (ab zeile 23 aufnehmen).
Zudem würde ich das "(Technologie-spezifischen)" entfernen.